client/lib/analytics/test/survicate.js

```diff
@@ -362,5 +384,62 @@ describe( 'survicate', () => {
 			addSurvicate();
 			expect( document.createElement ).not.toHaveBeenCalled();
 		} );
+
+		test( 'should set visitor traits when user is not on anonymous paths', () => {
```

Comment:
Should we also add integration tests that cover when the route is changed (from anonymous to non-anonymous for example), to ensure it loads correctly?